rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -83,19 +83,39 @@ rclcpp::QoS publisher_qos_for_topic(
   const auto offered_qos_profiles = profiles_yaml.as<std::vector<Rosbag2QoS>>();
   return Rosbag2QoS::adapt_offer_to_recorded_offers(topic.name, offered_qos_profiles);
 }
+
```

Comment:
```suggestion
```

Only will put this in one place for first pass review, please make sure not to commit any unrelated changes.